clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -102,7 +102,8 @@ public void testSerialization() throws Exception {
 
         checkSerialization(createUpdateMetadataRequest(0, null), 0);
         checkSerialization(createUpdateMetadataRequest(0, null).getErrorResponse(0, new UnknownServerException()), 0);
-        checkSerialization(createUpdateMetadataRequest(1, "rack1"), 1);
+        checkSerialization(createUpdateMetadataRequest(1, null), 1);
```

Comment:
Good point. The reason I wanted to test non-null rack in v1 serialization is to make sure that we can sue the same UpdateMetadataRequest constructor. But I overlooked that I already excluded rack in the above code.

I will fix this code and still include a rack to test v1 serialization. This can be done after merging your PR.